tools/manual_testing/plot.ipynb

```diff
@@ -159,7 +159,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "da.plot(labels={'xx':'xx2'}, errorbars=False)"
+    "da.plot(labels={'xx':'xx2'}, errorbars=False, resolution=3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7ef29ec-c530-4a88-ab21-cd0d1176c1ce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da.plot(labels={'xx':'xx2'}, errorbars=False, resolution=3, resampling_mode='sum')"
```

Comment:
Can you also add some tests in the python plotting tests? Thanks.

Reply:
Done.